lfs/lfs.go

```diff
@@ -29,6 +30,7 @@ var (
 	LocalGitStorageDir string // parent of objects/lfs (may be same as LocalGitDir but may not)
 	LocalMediaDir      string // root of lfs objects
 	LocalObjectTempDir string // where temporarily downloading objects are stored
+	LocalStorage       *localstorage.LocalStorage
```

Comment:
I find it a little confusing that the var is called the same as the type (and package), maybe that's just me but I'd prefer a different name for the var. Not sure what, StorageInst?